crates/engine/tree/benches/state_root_task.rs

```diff
@@ -12,7 +13,7 @@ use reth_provider::{
     test_utils::{create_test_provider_factory, MockNodeTypesWithDB},
     AccountReader, HashingWriter, ProviderFactory,
 };
-use reth_testing_utils::generators::{self, Rng};
+use reth_testing_utils::generators::Rng;
```

Comment:
can we do this now that we don't use anything from `reth_testing_utils::generators`?
```suggestion
use rand::Rng;
```